src/meta_memcache/serializer.py

```diff
@@ -40,20 +40,16 @@ def serialize(
             data = zlib.compress(data)
         return EncodedValue(data=data, encoding_id=encoding_id)
 
-    # pyre-ignore[3]
     def unserialize(self, data: Blob, encoding_id: int) -> Any:
         if encoding_id & self.ZLIB_COMPRESSED:
-            # pyre-ignore[6]
             data = zlib.decompress(data)
             encoding_id ^= self.ZLIB_COMPRESSED
 
         if encoding_id == self.STR:
             return bytes(data).decode()
         elif encoding_id in (self.INT, self.LONG):
-            # pyre-ignore[6]
             return int(data)
```

Comment:
Why mypy does not complain about this?

Reply:
Possibly because there are some type overlaps with the `int` constructor